### PR TITLE
WIP: refactor(head_ref): use joint trajectory action interface

### PR DESCRIPTION
**Pull request by rokusottervanger**

### CODE REVIEW

**Comment by rokusottervanger** [comment]
@reinzor , could you shed your light on this for a moment?

File: CMakeLists.txt.user [hunk 1]
```diff
@@ -0,0 +1,540 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (MatthijsBurgh) [inline CMakeLists.txt.user]: This file shouldn't be in the PR.

Comment (rokusottervanger) [reply CMakeLists.txt.user]: Oops, sorry about that. Should be added to .gitignore I guess.

**Comment by MatthijsBurgh** [comment]
Not using at all. Not important anymore.

**Pull request closed without merging**